contracts/v0.6/swap/ERC20TokenPeg.sol

```diff
@@ -0,0 +1,116 @@
+
+// SPDX-License-Identifier: MIT
+pragma solidity >0.6.0 <0.8.0;
+
+import "@openzeppelin/contracts/math/SafeMath.sol";
+import "@openzeppelin/contracts/proxy/Initializable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/SafeERC20.sol";
+import "./ISwap.sol";
+
+/**
+ * @title ERC20TokenPeg
+ * @notice Using this contract users can swap between pegged source and target tokens.
+ * @dev Make sure only to deploy with target ERC20 token having a fixed supply
+ * as this contract assumes a simple 1:1 fixed peg.
+ */
+contract ERC20TokenPeg is ISwap, Initializable {
+  using SafeMath for uint256;
+  using SafeERC20 for IERC20;
+
+  IERC20 private immutable _source;
```

Comment:
should this be `SOURCE` according to our style? (since it's immutable)

Reply:
I agree! As this is the case:

> The contract creation code generated by the compiler will modify the contract’s runtime code before it is returned by replacing all references to immutables with the values assigned to them. This is important if you are comparing the runtime code generated by the compiler with the one actually stored in the blockchain.
> -- Solidity docs

Meaning, any `immutable` var basically becomes a `constant` on deployment.

Reply:
We agreed to have an `_` prefix for internal & private methods, what about storage vars?

Reply:
storage vars look like this `s_storageVar`